src/cockpit/channels/stream.py

```diff
@@ -66,6 +66,7 @@ def create_transport(self, loop: asyncio.AbstractEventLoop, options: Dict[str, A
         cwd: Optional[str] = options.get('directory')
         pty: bool = options.get('pty', False)
         window: Dict[str, int] = options.get('window')
+        environ: Optional[List[str]] = options.get('environ')
```

Comment:
I've actually grown to dislike these sorts of type annotations.  I'd prefer you exclude it and let mypy complain.  This is a legitimate error that we'll need to address some day.

Reply:
I kinda dislike mixing styles though, but I'm fine with it.